lib/service_actor/checkable.rb

```diff
@@ -6,6 +6,15 @@ def self.included(base)
   end
 
   module PrependedMethods
+    CHECK_CLASSES = [
+      ServiceActor::Checks::TypeCheck,
+      ServiceActor::Checks::MustCheck,
+      ServiceActor::Checks::InclusionCheck,
+      ServiceActor::Checks::NilCheck,
+      ServiceActor::Checks::DefaultCheck
+    ].freeze
+    private_constant :CHECK_CLASSES
```

Comment:
We should probably remove the `check_classes` method then.

Reply:
Absolutely, I just forgot